docs/whatsnew/4.0.rst

```diff
@@ -15,7 +15,47 @@ SunPy 4.0 also includes a large number of smaller improvements and bug fixes, wh
 
 By the numbers:
 
-TODO: fill this in at release time.
+This release of sunpy contains 1310 commits in 182 merged pull requests closing 83 issues from 29 people, 19 of which are first-time contributors to sunpy.
+
+* 1310 commits have been added since 3.1
+* 83 issues have been closed since 3.1
+* 182 pull requests have been merged since 3.1
+* 29 people have contributed since 3.1
+* 19 of which are new contributors
+
+The people who have contributed to the code for this release are:
+
+-  Akash Verma  *
+-  Alasdair Wilson  *
+-  Albert Y. Shih
+-  Alex Kaszynski  *
+-  Alex W  *
+-  Andy Tang  *
+-  Anubhav Sinha  *
+-  Conor MacBride
+-  Daniel Garcia Briseno  *
+-  David Stansby
+-  Devansh Shukla  *
+-  Jan Gieseler  *
+-  Jeffrey Aaron Paul
+-  Jia Qing  *
+-  Karthikeyan Singaravelan  *
+-  Krish Agrawal  *
+-  Laura Hayes
+-  Lazar Zivadinovic
+-  Nabil Freij
+-  Noah Altunian  *
+-  Rohan Sharma  *
+-  Samriddhi Agarwal
+-  Stuart Mumford
+-  Thomas Braccia  *
+-  Tim Gates  *
+-  Will Barnes
+-  William Russell  *
+-  cbard  *
```

Comment:
Does this poor person not have a real name :stuck_out_tongue:

Reply:
@cbard Do you have a name you would like us to use for your contributions?

Reply:
It is true that my birth certificate doesn't say `cbard`... 😛 

You can put in Chris Bard. Could we also put in Raphael Attie in this list as well (if not already in)? His work was the impetus for my putting in a PR in the first place.

Reply:
Name updated, and Raphael added 👍